.github/ISSUE_TEMPLATE/performance-challenge-program.md

```diff
@@ -1,21 +1,25 @@
 ---
-name: "✈️ Performance Challenge Program"
-about: PCP Season1 Program
-title: 'PCP: '
-labels: ''
-assignees: ''
+name: "✈️ Performance Challenge Program (PingCAP Internal)"
+title: 'PCP-X: '
 
 ---
 
-## PCP Issue
+## Description
 
-- difficult: medium
-- score: 150
-- min score required: 100
-- expired: 7d
-- Recommended Skills:
-- Mentor(s):
+To be filled
 
-## Description
+## Difficulty
+
+* To be filled
+
+## Score 
+
+* To be filled
```

Comment:
em, what does Score mean?

Reply:
Looks like we should simply remove this template to avoid confusing other users.. they are not supposed to create such kind of issue.

Reply:
@winkyao What do you think